packages/rpc/src/rpc-request-coalescer.ts

```diff
@@ -1,9 +1,9 @@
-import type { RpcTransport } from '@solana/rpc-spec';
+import type { RpcResponse, RpcTransport } from '@solana/rpc-spec';
 
 type CoalescedRequest = {
     readonly abortController: AbortController;
     numConsumers: number;
-    readonly responsePromise: Promise<unknown>;
+    readonly responsePromise: Promise<RpcResponse | undefined>;
```

Comment:
Does this need to be `| undefined`?

Reply:
That's already the current type definition of the coalescer. It just didn't show before because `unknown | undefined` is equivalent to `unknown`.